apps/vendor-portal/src/pages/App.vue

```diff
@@ -234,7 +234,6 @@ const menuItems = reactive<IMenuItems[]>([
           readonly: true,
           query: {
             isPublished: true,
-            searchFromAllSellers: true,
```

Comment:
this is incorrect solution for given issue, by this change we limit the resulting products with only seller's products. This is equal `My  product` menu item